nginx/nginx.conf

```
@@ -1620,6 +1620,7 @@ http {
             rewrite "^/display/JENKINS/Jenkins\+CLI$" "https://www.jenkins.io/doc/book/managing/cli/" permanent;
             rewrite "^/display/JENKINS/Jenkins\+SSH$" "https://www.jenkins.io/doc/book/managing/cli/" permanent;
             rewrite "^/display/JENKINS/Dependencies\+among\+plugins" "https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/" permanent;
+            rewrite "^/display/JENKINS/Building\+Jenkins" "https://www.jenkins.io/doc/developer/building/#building-and-debugging/" permanent;
```

I think that we should redirect to the core contributing file, not to the stub on www.jenkins.io
```suggestion
            rewrite "^/display/JENKINS/Building\+Jenkins" "https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md" permanent;
```